src/components/Heading/Heading.tsx

```diff
@@ -12,8 +12,14 @@ export interface HeadingProps {
   element?: HeadingTagName;
   /** The content to display inside the heading */
   children?: React.ReactNode;
+  /** A unique identifier for the heading, used for reference in anchor links  */
+  id?: string;
```

Comment:
@kyledurand Turns out the Heading didn't take an ID prop, so I updated the component and type docs.
If this looks good, let me know if there are any examples we should add to the README / code sample docs